addons/Pouches/Pouches.lua

```diff
@@ -51,11 +54,21 @@ end)
 function use_item()
     windower.chat.input('/item "' .. item.name .. '" <me>')
     item.count = item.count - 1
-    if item.count > 0 and windower.ffxi.get_player().status == 0 then
-        use_item:schedule(item.delay)
+    if item.count > 0 and windower.ffxi.get_player().status == 0 then	
+        if inventory_info.current < inventory_info.max then 
```

Comment:
I think it makes most sense to use this place to query for the current values for `current` and `max`. This would eliminate all other instances of this. Also, you should call the function only once:
```lua
local info = windower.ffxi.get_bag_info(0)
if info.count < info.max then
    -- do eet
else
    -- don't... do eet
end
```